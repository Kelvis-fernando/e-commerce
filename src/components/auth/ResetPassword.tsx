import { AuthModalContainer } from "../../styles/auth/loginModal";
import { ForgotPasswordProps } from "../../types/auth";
import TitleAndSubtitle from "../layout/TitleAndSubtitle";

const ResetPassword = ({ onBack }: ForgotPasswordProps) => {
  return (
    <AuthModalContainer>
      <TitleAndSubtitle
        title="Altere a sua senha"
        subtitle="Adicione e confirme a sua nova senha"
      />
      <form>
        <label htmlFor="password">Senha</label>
        <input type="password" name="password" id="password" />
        <label htmlFor="confirmPassword">Confirme a senha</label>
        <input type="password" name="confirmPassword" id="confirmPassword" />
        <button type="submit">Alterar</button>
      </form>
      <h2
        onClick={() => {
          if (onBack) return onBack(false);
        }}
      >
        Voltar
      </h2>
    </AuthModalContainer>
  );
};

export default ResetPassword;
