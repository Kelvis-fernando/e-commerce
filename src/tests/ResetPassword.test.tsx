import React from "react";
import { render } from "@testing-library/react";
import ResetPassword from "../components/auth/ResetPassword";

describe("ResetPassword", () => {
  test("should render and match snapshot", () => {
    const { container } = render(<ResetPassword />);

    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
