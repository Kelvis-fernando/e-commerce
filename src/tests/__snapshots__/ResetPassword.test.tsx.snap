// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPassword should render and match snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe hlQNyq"
  >
    <div
      class="sc-gswNZR dKCtgi"
    >
      <h1>
        Altere a sua senha
      </h1>
      <h3>
        Adicione e confirme a sua nova senha
      </h3>
    </div>
    <form>
      <label
        for="password"
      >
        Senha
      </label>
      <input
        id="password"
        name="password"
        type="password"
      />
      <label
        for="confirmPassword"
      >
        Confirme a senha
      </label>
      <input
        id="confirmPassword"
        name="confirmPassword"
        type="password"
      />
      <button
        type="submit"
      >
        Alterar
      </button>
    </form>
    <h2>
      Voltar
    </h2>
  </div>
</div>
`;
